src/table/actionHeaderFactory.js

```diff
@@ -69,6 +69,17 @@ function actionFactory(action, actionConfiguration, schema) {
         onChange(newFormData);
       }
     };
+  } else if (action === "inheritedAction") {
+    return (rowIndex, formData, inheritedActionName, onChange) => {
+      if (inheritedActionName === "delete") {
+        let newFormData = formData.slice(0);
+        newFormData.splice(rowIndex, 1);
+        onChange(newFormData);
+      } else {
+        //Edit
+        window.handleCptEditPopUp(rowIndex);
```

Comment:
If we go this route, we will need to check if the function exists before we call it.

Reply:
Added the check for the function exists